content/en/Integrations/IntegrationBuilder/How to Guides/add-pentest-title-to-jira-issue-summary.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "Add Pentest Title to Jira Issue Summary"
+linkTitle: "Add Pentest Title to Jira Issue Summary"
```

Comment:
I would use update, modify, adjust instead of add. Add would mean something different for me as a first-team reader of this page; it could be a too-engineering way of thinking.